telco-ran/configuration/source-crs/node-tuning-operator/TunedPerformancePatch.yaml

```diff
@@ -1,32 +1,37 @@
 apiVersion: tuned.openshift.io/v1
 kind: Tuned
 metadata:
-  name: performance-patch
+  name: ran-du-performance
   namespace: openshift-cluster-node-tuning-operator
   annotations:
     ran.openshift.io/ztp-deploy-wave: "10"
 spec:
   profile:
-    - name: performance-patch
+    - name: ran-du-performance
       # Please note:
       # - The 'include' line must match the associated PerformanceProfile name, following below pattern
       #   include=openshift-node-performance-${PerformanceProfile.metadata.name}
-      # When using the standard (non-realtime) kernel, disable timer migration by adding the following in the data section:
-      # [sysctl]
-      # kernel.timer_migration=0
       data: |
         [main]
         summary=Configuration changes profile inherited from performance created tuned
-        include=openshift-node-performance-openshift-node-performance-profile
+        include=openshift-node-performance-openshift-node-performance-profile,performance-patch-${f:lscpu_check:Vendor ID\:\s*GenuineIntel:intel:Vendor ID\:\s*AuthenticAMD:amd:Architecture\:\s*aarch64:arm}-${f:lscpu_check:Architecture\:\s*x86_64:x86:Architecture\:\s*aarch64:aarch64}
```

Comment:
Furthermore, is this lscpu mechanism stable?  It seems to depend on parsing a longer string than something simpler like `uname -m`.